framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -113,38 +124,35 @@ public ValueTransfer(CFAbstractAnalysis<CFValue, CFStore, CFTransfer> analysis)
         return ValueAnnotatedTypeFactory.getCharValues(intAnno);
     }
 
+    /** Returns a list of possible values, or null if no estimate is available. */
```

Comment:
I changed the documentation to clarify:
"Returns a list of possible values, or null if no estimate is available and any value is possible."
Does that make sense now?